metis-authentication/metis-authentication-rest/manifest.yml

```diff
@@ -1,10 +1,18 @@
 ---
 applications:
-- name: metis-authentication-rest-test-compose
-  memory: 2048M
-  instances: 1
-  host: metis-authentication-rest-test-compose
-  path: target/metis-authentication-rest-1.0-SNAPSHOT.war
-  buildpack: https://github.com/europeana/java-buildpack.git#feat/java8
-  services:
-    - metis-authentication-test-psql-compose
+  - path: target/metis-authentication-rest-9-SNAPSHOT.jar
+    name: metis-authentication-rest-test
+    host: metis-authentication-rest-test
+    stack: cflinuxfs3
+    buildpack: https://github.com/cloudfoundry/java-buildpack
```

Comment:
Double check if this apply: 
according to documentation this attribute `buildpack` is deprecated. They favor the use of `buildpacks`
example
```
   buildpacks:
   - https://github.com/cloudfoundry/java-buildpack
```

Reply:
Good point, I didn't know.
But I'll leave it as is for now, we will migrate very soon anyway.